File: stepik-java-api/src/main/java/org/stepik/api/actions/StepikRecommendationsAction.java
package org.stepik.api.actions;

import org.jetbrains.annotations.NotNull;
import org.stepik.api.client.StepikApiClient;
import org.stepik.api.queries.recommendations.StepikRecommendationsGetQuery;

/**
 * @author meanmail
 */
public class StepikRecommendationsAction extends StepikAbstractAction {
    public StepikRecommendationsAction(@NotNull StepikApiClient stepikApiClient) {
        super(stepikApiClient);
    }

    @NotNull
    public StepikRecommendationsGetQuery get() {
        return new StepikRecommendationsGetQuery(this);
    }
}

File: stepik-java-api/src/main/kotlin/org/stepik/api/actions/StepikRecommendationsAction.kt
package org.stepik.api.actions

import org.stepik.api.client.StepikApiClient
import org.stepik.api.queries.recommendations.StepikRecommendationsGetQuery

class StepikRecommendationsAction(stepikApiClient: StepikApiClient) : StepikAbstractAction(stepikApiClient) {
    
    fun get(): StepikRecommendationsGetQuery {
        return StepikRecommendationsGetQuery(this)
    }
}
